doc/manuals/user/service_path.md

```diff
@@ -85,3 +87,20 @@ Some additional remarks:
 -   Current version doesn’t allow to change the scope to which an entity
     belongs through the API (a workaround is to modify the
     \_id.servicePath field in the entities collection directly).
+
+## Service paths in subscriptions and registrations
+
+While entities belong to services *and* servicepaths, subscriptions and registrations
+belong *only* to the service. The servicepath in subscriptions and registrations
+doesn't denotes sense of beloging, but is the expression of the query associated
```

Comment:
typos: doesn't denote sense of belonging, but ...

Reply:
Fixed in 311d6ce61a3ca7d1d7be137647b9f58b2cb983b7